tests/on_target/utils/test_uart.py

```diff
@@ -105,3 +105,54 @@ def test_wait_ordered_7_none(time_sleep, time_time):
     with pytest.raises(AssertionError) as ex_info:
         u.wait_for_str_ordered(["abc", "def", "ghi", "jkl"], timeout=2)
     assert "abc missing" in str(ex_info.value)
+
+@patch("time.time", side_effect=counter())
+@patch("time.sleep")
+def test_wait_8_get_current_size(time_sleep, time_time):
+    u = mocked_uart()
+    u.log = "foo123\nbar123\nbaz123\n"
+    current_log_size = u.wait_for_str(["bar"], timeout=3)
```

Comment:
Did you intend to run `u.get_size()` here?

Reply:
I intended to check that u.wait_for_str returns the log size. But maybe we should have a test for get_size as well?